### ISSUE TITLE
[Bug?]: Can't customize Storybook!

### ISSUE THREAD

**Issue by theoradicella** [redwoodjs/redwood#6854]
### What's not working?

I wan't to disable essential addons from storybook but since Redwood merges it's own config, (which includes essential addons) I'm not able to do it.

Here I have opened the issue in the community: [https://opensource.adobe.com/spectrum-web-components/storybook/?path=/story/accordion--default](https://opensource.adobe.com/spectrum-web-components/storybook/?path=/story/accordion--default)

### Are you interested in working on this?

- [ ] I'm interested in working on this

**Comment by simoncrypta** [comment]
@theoradicella That's true that Redwood have an opinionated base Storybook config, maybe we should have a way to overwrite the config. What do you think @virtuoushub ?

**Comment by theoradicella** [comment]
That would be really useful. We are not using essential addons and we would really like to have our storybook clean. I'll be waiting for this. Thanks!
